janus_core/md.py

```diff
@@ -800,7 +800,7 @@ def get_log_header() -> str:
             Header for molecular dynamics log.
         """
         log_header = MolecularDynamics.get_log_header()
-        return log_header + " | T [K]"
+        return log_header + " | T* [K]"
```

Comment:
Is this a standard way to represent the target temperature?

Reply:
there is no standard, we have freedom

Reply:
Would it be better just be explicit then - `target T` or something, which also would also simplify getting the headers?

Reply:
in fairness probably we do not need the target at all in that file, since in theory it is in the log file anyhow... maybe is better to get rid of it alltogether, same for pressure.

Reply:
I suppose right now it doesn't serve much purpose, but when we reintroduce the temperature ramp, it can be useful to have them next to each other rather than having to figure out at which steps it changes. Arguably even then it may not be worth the extra IO though.

Reply:
good point! T_target shall be, same for pressure, p_target

Reply:
since this is a bigger pr that what I did maybe you prepare a separate one dealing with this and then I merge it in this?